USER_GUIDE.md

```diff
@@ -201,3 +216,68 @@ func main() {
 	log.Println("PING OK")
 }
 ```
+
+#### AWS SDK V2
+
+```go
+package main
```

Comment:
Can we keep the code formatting (like tabs/spaces) for all the examples in this file similar?

Reply:
I've reformatted the whole document with standard GoLand formatting options. Hope it's okay.